modules/auxiliary/gather/behind_cloudflare.rb

```diff
@@ -0,0 +1,445 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
```

Comment:
`https` rather than `http`

Reply:
Hello, sorry for the time to answer ... I'm overwhelmed.

I modified.